### PR TITLE
Clean/clang format upstream

### PR DESCRIPTION
**Pull request by marijnvdwerf**

### CODE REVIEW

File: include/items.h [hunk 1]
```diff
@@ -1,388 +1,389 @@
 #ifndef GUARD_ITEMS_H
 #define GUARD_ITEMS_H
 
-enum {
```
Comment (yenatch) [inline include/items.h]: weird (bad)

File: include/rtc.h [hunk 2]
```diff
@@ -3,19 +3,19 @@
 
 #include "siirtc.h"
 
-#define RTC_INIT_ERROR         0x0001
```
Comment (marijnvdwerf) [inline include/rtc.h]: Add guard

File: include/pokemon.h [hunk 3]
```diff
@@ -1,125 +1,126 @@
 #ifndef GUARD_POKEMON_H
 #define GUARD_POKEMON_H
 
-#define MON_DATA_PERSONALITY        0
```
Comment (marijnvdwerf) [inline include/pokemon.h]: add guard

File: include/items.h (hunk 1 above)
Comment (marijnvdwerf) [inline include/items.h]: What specific part are you talking about?

Comment (yenatch) [inline include/items.h]: It aligns comments to each block of comments but it should align to the whole block of code.

**Pull request closed without merging**